src/ui/DNA/shared/WideScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { useSizeStore } from 'src/ui/Onboarding/useSizeStore';
+import { useWindowSizeStore } from 'src/ui/shared/useWindowSizeStore';
```

Comment:
I'm not sure that we need to rename this hook in this pr, but ok

Reply:
Yeah, it's used everywhere in backup and onboarding, like on almost every page:
```
const { isNarrowView } = useWindowSizeStore();
```
So I decided to update it now (sorry if this change caused extra review work)